<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="QLineEdit" name="hostName_editBox">
          <property name="minimumSize">
           <size>
            <width>171</width>
            <height>20</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>host name</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QToolButton" name="pathDialog_button">
          <property name="minimumSize">
           <size>
            <width>41</width>
            <height>21</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>41</width>
            <height>22</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLineEdit" name="setupPy_EditBox">
          <property name="minimumSize">
           <size>
            <width>171</width>
            <height>20</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>setup.py filepath</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>setup.py filepath</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="port_editbox">
          <property name="minimumSize">
           <size>
            <width>41</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>41</width>
            <height>20</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>port</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>host name</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>port</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" rowspan="4">
         <widget class="QGroupBox" name="groupBox">
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>100</height>
           </size>
          </property>
          <property name="title">
           <string>property</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="3" column="0" colspan="3">
            <widget class="QPushButton" name="versionRefresh_button">
             <property name="text">
              <string>version refresh</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QLineEdit" name="trivialVersion_editBox">
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="minorVersion_editBox">
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLineEdit" name="majorVersion_editBox">
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="3">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Major/Minor/Trivial</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="upload_button">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>61</height>
         </size>
        </property>
        <property name="text">
         <string>upload</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>output</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="output_editBox">
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>541</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>hostName_editBox</tabstop>
  <tabstop>port_editbox</tabstop>
  <tabstop>setupPy_EditBox</tabstop>
  <tabstop>pathDialog_button</tabstop>
  <tabstop>majorVersion_editBox</tabstop>
  <tabstop>minorVersion_editBox</tabstop>
  <tabstop>trivialVersion_editBox</tabstop>
  <tabstop>versionRefresh_button</tabstop>
  <tabstop>upload_button</tabstop>
  <tabstop>output_editBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
